NetCasbin.UnitTest/EnforcerTest.cs

```diff
@@ -27,11 +28,11 @@ public EnforcerTest(ITestOutputHelper testOutputHelper, TestModelFixture testMod
         [Fact]
         public void TestKeyMatchModelInMemory()
         {
-            var m = Model.Model.CreateDefault();
-            m.AddDef("r", "r", "sub, obj, act");
-            m.AddDef("p", "p", "sub, obj, act");
-            m.AddDef("e", "e", "some(where (p.eft == allow))");
-            m.AddDef("m", "m", "r.sub == p.sub && keyMatch(r.obj, p.obj) && regexMatch(r.act, p.act)");
+            var m = DefaultModel.Create();
+            m.AddDefinition("r", "r", "sub, obj, act");
```

Comment:
Why change this? I think the new name is not our API: https://casbin.org/docs/en/model-storage#load-model-from-code

Reply:
Because this is more in line with the naming style of .NET, but it is more important to keep a unique name if this is a standard API.